flightrl/rpg_baselines/ppo/ppo2.py

```diff
@@ -430,6 +434,11 @@ def learn(self, total_timesteps, log_dir, logger,
                         # compatibility with callbacks that have no return statement.
                         if callback(locals(), globals()) is False:
                             break
+                    
+                    # save weights every 5 updates
+                    if update % 5 == 0:
```

Comment:
Overall this looks good, thanks! One improvement that would be very helpful is if you could make the number 5 a parameter (save weight every n updates) and have this be an optional argument in run_drone_control when starting the training. If no number is passed then the weights will not be saved during training and only at the end